src/pages/talks/[talk].astro

```diff
@@ -0,0 +1,195 @@
+---
+import { FavoriteButton } from "../../components/button/favoriteButton/FavoriteButton";
+import TalkLayout from "../../layouts/Layout.astro";
+import {
+  getDisciplineColorPair,
+  getDisciplineName,
+} from "../../utils/discipline";
+import { schedule, type Talk } from "../../utils/schedule";
+
+const exampleDescription =
+  "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.";
+const exampleSpeakerDescription =
+  "Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto beatae vitae dicta sunt explicabo";
+
+function getTalkById(talkId: string): Talk | undefined {
+  const allTalks: Talk[] = schedule
+    .flatMap((block) => block.tracks.flat())
+    .filter((scheduleEntry) => scheduleEntry.type === "talk");
+
+  return allTalks.find((t) => t.id === talkId);
+}
```

Comment:
Kan kanskje flytte funksjonen til en annen fil? Ha den i schedule eller en mappe for schedule med typer, utils...

Reply:
Er ikke kritisk å få på plass nå. Har tenkt på det som en faktoreringsoppgave